modules/reactor/doc/content/source/meshgenerators/SubdomainExtraElementIDGenerator.md

```diff
@@ -4,9 +4,14 @@
 
 ## Overview
 
-The `SubdomainExtraElementIDGenerator` assigns element IDs based on mesh subdomain IDs of an input mesh.
-If the element IDs do not exist in the input mesh, they will be added by this mesh generator.
+The `SubdomainExtraElementIDGenerator` assigns extra element IDs based on mesh subdomain IDs of an input mesh.
+If the extra element IDs do not exist in the input mesh, this mesh generator will add them.
+The [!param](/Mesh/SubdomainExtraElementIDGenerator/subdomains) parameter can accept both subdomain names and subdomain IDs.
+The [!param](/Mesh/SubdomainExtraElementIDGenerator/extra_element_ids) parameter is a two-dimensional vector separated by semicolon and then white space as normal MOOSE input syntax for two-dimensional vectors.
+Its first dimensitionality must be equal to the size of the [!param](/Mesh/SubdomainExtraElementIDGenerator/extra_element_id_names) parameter and the size of each sub-vector must be equal to the size of the [!param](/Mesh/SubdomainExtraElementIDGenerator/subdomains) parameter.
+
 Default element IDs can be provided through [!param](/Mesh/SubdomainExtraElementIDGenerator/default_extra_element_ids) for elements in subdomains that are not specified in the [!param](/Mesh/SubdomainExtraElementIDGenerator/subdomains) parameter.
+Because of this, [!param](/Mesh/SubdomainExtraElementIDGenerator/default_extra_element_ids) is active only when the [!param](/Mesh/SubdomainExtraElementIDGenerator/subdomains) parameter does not include all mesh subdomains.
```

Comment:
this is already implied

Reply:
Yes. But this sentence makes things clearer to users. They do not need to do the speculation.